packages/utils/src/RBAC.js

```diff
@@ -0,0 +1,93 @@
+
+export async function getRBAC(applicationName = '') {
+    const insights = window.insights;
+    if (
+        insights &&
+    insights.chrome &&
+    insights.chrome.auth &&
+    insights.chrome.auth.getUser &&
+    insights.chrome.getUserPermissions
+    ) {
+        const user = await insights.chrome.auth.getUser();
+        const permissions = await insights.chrome.getUserPermissions(applicationName);
+        return {
+            isOrgAdmin: user.identity.user.is_org_admin,
+            permissions
+        };
+    }
+
+    return { isOrgAdmin: false, permissions: null };
+}
+
+export function doesHavePermissions(RBACResults, permissionList) {
+    const { isOrgAdmin, permissions } = RBACResults;
+    if (isOrgAdmin) {
```

Comment:
This is not quite correct, if user is Org admin they might not have access to something based on the permission.